addons/mail/static/src/models/message/message.js

```diff
@@ -107,7 +107,10 @@ registerModel({
                 }
             }
             if ('partner_ids' in data && this.messaging.currentPartner) {
-                data2.isCurrentPartnerMentioned = data.partner_ids.includes(this.messaging.currentPartner.id);
+                data2.recipients = insertAndReplace(data.partner_ids.map(partner_id => ({ id: partner_id })));
```

Comment:
As this is master you can remove the old `partner_ids` key (python, js, mock, ...).